force-app/main/default/lwc/futureSessionDeleter/futureSessionDeleter.js

```diff
@@ -0,0 +1,75 @@
+import { LightningElement, api } from "lwc";
+import { format, handleError } from "c/util";
+import { ShowToastEvent } from "lightning/platformShowToastEvent";
+import deleteSessionsAfter from "@salesforce/apex/ServiceScheduleCreatorController.deleteSessionsAfter";
+
+import HEADER_LABEL from "@salesforce/label/c.Delete_Future_Sessions";
+import MESSAGE_LABEL from "@salesforce/label/c.Delete_Sessions_After";
+import DELETE from "@salesforce/label/c.Delete";
+import CANCEL from "@salesforce/label/c.Cancel";
+import SUCCESS_LABEL from "@salesforce/label/c.Success";
+import DELETE_SUCCESS from "@salesforce/label/c.Delete_Sessions_Success";
+import INVALID_DATE from "@salesforce/label/c.Delete_Sessions_Invalid_Date";
+
+export default class FutureSessionDeleter extends LightningElement {
+    @api recordId;
+    startDate = new Date().toISOString();
```

Comment:
Admittedly this would be pretty edge case but since we are using the browser and everything is using the Salesforce User locale we could run into an issue where the dates do not match. Especially late at night / early morning or if we are dealing with sessions in a time-zone that crosses days and the browser is set to a different locale like in the attached Brisbane example:
![Screen Shot 2020-10-07 at 9 06 24 AM](https://user-images.githubusercontent.com/450473/95342136-b1fc4900-087c-11eb-8b26-2a372e73b802.png)
![Screen Shot 2020-10-07 at 9 06 00 AM](https://user-images.githubusercontent.com/450473/95342140-b294df80-087c-11eb-83fd-053a34207975.png)